examples/media/src/app.css

```diff
@@ -591,6 +575,49 @@ body.filtering {
   display: none;
 }
 
+/* Footer */
+
+.footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.back-to-top {
+  margin: 0 2.5rem 1rem auto;
+  height: 2.5rem;
+  width: 2.5rem;
+}
+
+.filtering .back-to-top {
+  display: none;
+}
+
+.mentions {
+  padding: 0.5rem 1rem;
+  font-size: 0.75rem;
```

Comment:
Could we make the text slightly bigger? I was thinking maybe the same size as the summary content in each hit.